app/models/case/base.rb

```diff
@@ -554,7 +557,7 @@ def responded_late?
 
   def responded_in_time?
     return false unless closed_for_reporting_purposes?
-    date_responded <= external_deadline
+    date_responded.nil? ? false : date_responded <= external_deadline
```

Comment:
Its rare I think to have expressions in ternary operators - I think it may be clearer to just add another guard clause:

```ruby
return false if date_responded.nil?
date_responded <= external_deadline
```